plug-ins/prometheus/test-setup.md

```diff
@@ -0,0 +1,32 @@
+# Prometheus Plugin Test Setup
+
+* Install Prometeus 
+  * wget https://github.com/prometheus/prometheus/releases/download/v2.42.0/prometheus-2.42.0.linux-amd64.tar.gz
+  * tar xvzf prometeus-2.4.2.0.linux-amd64.tar.gz
```

Comment:
If no minimum version or the packaged versions meet the minimum, include `apt install` instructions for the packaged versions, with package names as appropriate.